### PR TITLE
Gui changes for anonimity

### PR DESCRIPTION
**Pull request by rjruigrok**
* Hops slider moved from save-as to settings panel
* Removed timeout switch setting in favor of download policies from save-as dialog
* Implemented proposed changes from #1390 and #1393

### CODE REVIEW

**Comment by synctext** [comment]
retest this please

**Comment by synctext** [comment]
retest this please

**Comment by rjruigrok** [comment]
Test passes, just some remote search failures. :)

This PR is ready. 
I had a look into moving the fetching-torrent status (which is nicer, but current solution also works fine), but the whole downloading / fetching / adding is really complex code. I need more time for this part, so it will be in a next PR.

**Comment by rjruigrok** [comment]
retest this please

**Comment by synctext** [comment]
retest this please

**Comment by synctext** [comment]
retest this please

**Comment by synctext** [comment]
retest this please

**Comment by whirm** [comment]
@rjruigrok looks fine to me, @LipuFei you can merge it if you have nothing to add.

File: Tribler/Main/vwxGUI/__init__.py [hunk 1]
```diff
@@ -53,6 +53,10 @@
 SEPARATOR_GREY = wx.Colour(210, 210, 210)
 FILTER_GREY = wx.Colour(240, 240, 240)
 
+DOWNLOAD_POLICY_HIDDEN_120_FALLBACK = 0
```
Comment (LipuFei) [inline Tribler/Main/vwxGUI/__init__.py]: I think it's more appropriate to move those into `Tribler/Core`, for example `Tribler/Core/simpledef.py`.

**Comment by LipuFei** [comment]
@rjruigrok Check the comment. The rest looks good.

**Comment by rjruigrok** [comment]
Nice. Let's wait for the test result. :+1:

**Comment by rjruigrok** [comment]
retest this please

**Comment by rjruigrok** [comment]
retest this please

**Comment by rjruigrok** [comment]
retest this please

**Pull request merged**